doc/intl.md

```diff
@@ -0,0 +1,192 @@
+- New functions or properties in the Intl object prototypes (for e.g. Intl.Collator.prototype.compare)
+- New locale aware functions in standard Javascript object prototypes (for e.g. String.prototype.localeCompare)
+
+One standard internationalization implementation strategy followed by other engines, is to bundle a library and data (typically [icu](http://site.icu-project.org/)) along with the application package. This guarantees deterministic functionality at the cost of applications package bloat.  We decided to consume the Android platform libraries and data for space efficiency, but at the cost of some variance in behaviour across  Android platforms.
+
+# ECMA-402 Compliance
+
+## Supported
+
+- Intl.Collator
+  - Intl.Collator.supportedLocalesOf
+  - Intl.Collator.prototype.compare
+  - Intl.Collator.prototype.resolvedOptions
+
+- Intl.NumberFormat
+  - Intl.NumberFormat.supportedLocalesOf
+  - Intl.NumberFormat.prototype.format
+  - Intl.NumberFormat.prototype.formatToParts
+  - Intl.NumberFormat.prototype.resolvedOptions
+
+- Intl.DateTimeFormat
+  - Intl.DateTimeFormat.supportedLocalesOf
+  - Intl.DateTimeFormat.prototype.format
+  - Intl.DateTimeFormat.prototype.formatToParts
+  - Intl.DateTimeFormat.prototype.resolvedOptions
+
+- Intl.getCanonicalLocales
+
+- String.prototype
+  - localeCompare
+  - toLocaleLowerCase
+  - toLocaleUpperCase
+
+- Array.prototype
+  - toLocaleString
+
+- Number.prototype
+  - toLocaleString
+
+- Date.prototype
+  - toLocaleString
+  - toLocaleDateString
+  - toLocaleTimeString
+
+## Not yet supported
+
+- [Intl.PluralRules](https://tc39.es/ecma402/#pluralrules-objects)
+
+- [Intl.RelativeTimeFormat](https://tc39.es/ecma402/#relativetimeformat-objects)
+
+- [Intl.DisplayNames](https://tc39.es/proposal-intl-displaynames/#sec-intl-displaynames-constructor)
+
+- [Intl.ListFormat](https://tc39.es/proposal-intl-list-format/#sec-intl-listformat-constructor)
+
+- [Intl.Locale](https://tc39.es/ecma402/#sec-intl-locale-constructor)
+
+- Intl.DateTimeFormat properties
+   - [dateStyle/timeStyle](https://tc39.es/proposal-intl-datetime-style/)
+   - [dayPeriod](https://github.com/tc39/ecma402/issues/29)
+   - [fractionalSecondDigits](https://github.com/tc39/ecma402/pull/347)
+- [BigInt.prototype.toLocaleString](https://tc39.es/ecma402/#sup-bigint.prototype.tolocalestring)
+
+## Excluded
+
+- [formatMatcher](https://tc39.es/ecma402/#sec-basicformatmatcher) property in Intl.DateTimeFormat.
+
+## Known Issues
+
+### Android 11
+
+- The keys of the object returned by &#39;resolvedOptions&#39; function in all Intl services are not deterministically ordered as prescribed by spec.
+- DateFormat: ECMAScript [beginning of time](https://www.ecma-international.org/ecma-262/11.0/index.html#sec-time-values-and-time-range) (-8,640,000,000,000,000), is formatted as November 271817, instead of expected April 271822.

```

Comment:
It's very important to properly format this sentence :) I'm assuming `April 271822` is the function output? They should be `-quoted